src/hb-ot-glyf-table.hh

```diff
@@ -275,16 +317,19 @@ struct glyf
 						 composite);
     }
 
+    static const int FLAG_ON_CURVE = 0x01;
```

Comment:
Please use (anonymous) enum instead of "static const int". We're not Java :D.

Reply:
:D cc: @garretrieger